x-pack/plugins/ml/public/jobs/new_job/simple/single_metric/create_job/create_job.html

```diff
@@ -133,8 +160,17 @@ <h2 class="euiTitle euiTitle"><i class='fa fa-spinner fa-spin'></i></h2>
             </div>
             <div class='status-label' ng-show="chartState === CHART_STATE.NO_RESULTS">
               <div class='no-results'>
-                <h4 class="euiTitle euiTitle--small"><i class="fa fa-info-circle"></i> No results found</h4>
-                <span>Consider using the <a ng-click="setFullTimeRange()">full {{ui.indexPattern.title}} data</a></span>
+                <h4
+                  class="euiTitle euiTitle--small"
+                  i18n-id="xpack.ml.newJob.simple.singleMetric.noResultsFoundTitle"
+                  i18n-default-message="{infoIcon} No results found"
+                  i18n-values="{ html_infoIcon: '<i class=\'fa fa-info-circle\'></i>' }"
+                ></h4>
+                <span
+                  i18n-id="xpack.ml.newJob.simple.singleMetric.noResultsFoundDescription"
+                  i18n-default-message="Consider using the {indexPatternLink}"
+                  i18n-values="{ html_indexPatternLink: '<a ng-click=\'setFullTimeRange()\'>' + indexPatternLinkText + '</a>' }"
```

Comment:
The link doesn't work because 'ng-click' is not compiled.

Reply:
fixed

Reply:
Please add the description as well.

Reply:
done